zaldis/PhonesBook
# Update project's structure

- Move communication logic into the `handlers` package
- Move useful func into the `utils` package

## HOW TO TEST

1. Prepare `phones.json` file with some contacts.
1. Run `python main.py` script.
1. Check via the `get` command that all contacts are loaded correctly.
1. Add new contacs via the `set` command.
1. Check that contacts are added properly.
1. Exit the program.
1. Check that all contacts are saved properly into the phones.json file.
